lib/tasks/reporting.rake

```diff
@@ -1,85 +1,6 @@
 require "ruby-progressbar"
 
 namespace :reporting do
-  def opts_from_environment(*option_keys)
```

Comment:
I'd be tempted to disable the cop as you've suggested, rather than remove these.

Reply:
I agree, it's not ideal to remove functionality in a 'linting' PR. On the other hand, it looks like this file is requiring more effort to keep linted than the value it delivers (none). I think it's time to directly tackle the real issue: this code isn't delivering any value.

Options:

- Remove it as part of this PR
- Remove it in a prerequisite PR

What do you think?

Reply:
They look quite useful, which is why I hesitated, but I've just had a search in Zendesk to see if these are mentioned anywhere, and I can't see that it's been used. So I'm happy for us to just remove them. 👍